contracts/InteractionNFT.sol

```diff
@@ -0,0 +1,32 @@
+
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.6.10;
+pragma experimental ABIEncoderV2;
+
+import "@openzeppelin/contracts/token/ERC1155/ERC1155Burnable.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
+
+contract InteractionNFT is ERC1155Burnable {
+    using Counters for Counters.Counter;
+    Counters.Counter interactionId;
+
+    uint public constant ROLE1 = 0;
+    uint public constant ROLE2 = 1;
+    uint public constant ROLE3 = 2;
```

Comment:
Why not use enum? Ideally we should have unified datatypes lib

Reply:
I agree. Let's create it post hackathon cause we'll be using it in all the products and now it's not the right time to redeploy everything we have :D